src/vm/gccover.h

```diff
@@ -108,6 +107,72 @@ class GCCoverageInfo {
 
 #endif // _TARGET_*
 
+inline bool IsGcCoverageInterruptInstructionVal(UINT32 instrVal)
```

Comment:
What is the reason for moving the implementation into the header file?

Reply:
The DAC relies upon this method, And I get an unsat symbol for the implementation of it when building: `mscordaccore.dll`

Reply:
OK, that seems reasonable though unexpected. It might be worth a comment to that effect (i.e. that, although gccover.h is needed to build mscordaccore.dll it doesn't include gccover.cpp.